### PR TITLE
RFC24: add cheque info

### PR DESCRIPTION
**Pull request by rev-chaos**
You can send and claim a cheque cell in the Lina by [mercury](https://github.com/nervosnetwork/mercury) with mainnet configuration with the rightful cheque settings written in this RFC.

A theoretical way to verify:
1. The code_hash of cheque script is `0x00000000000000000000000000000000000000000000000000545950455f4944`. See the cheque deploy transaction for [mainnet](https://explorer.nervos.org/transaction/0x0a34aeea122d9795e06e185746a92e88bca0ad41b0e5842a960e5fd1d43760a6) and [testnet](https://explorer.nervos.org/aggron/transaction/0x1b16769dc508c8349803fe65558f49aa8cf04ca495fbead42513e69e46608b6c). The deploy info is also written in this PR.
So, the binary code of cheque is consistent between mainnet and testnet.
2. The data of dep_group cell is completely obeying the dep group rules.
3. We have fully tested the cheque deploying ones on the testnet by mercury. An [example](https://explorer.nervos.org/aggron/transaction/0xed711c1354609792e9db66ce23653d07d00fb5d5c17e892b834025c53b76f227) for create cheque cell. An [example](https://explorer.nervos.org/aggron/transaction/0xe665735a28cbaefa501924f489e3464c145b7885fdfb709ef76b56e2620329ad) for claim cheque cell. 
So, according to theoretical derivation, we can make sure the cheque dep_group cell in mainnet can work.

A practical way to verify:  Use ckb-sdk v0.43.1 or above to successfully create a cheque cell.
1. Get some CKB and sUDT for test. CKB for test should above 305.
2. Set mainnet mercury URL in ckb-sdk, [java](https://github.com/nervosnetwork/ckb-sdk-java/blob/455f866c6faf5f63c781e74edefac5f7426a4019/ckb-mercury-sdk/src/test/java/mercury/constant/MercuryApiFactory.java#L8) or [go](https://github.com/nervosnetwork/ckb-sdk-go/blob/113ffa36146b34f6c294633a41ed961f0aaba1d7/mercury/example/constant/mercury_api_factory.go#L7) . We have deployed a public server here (https://mercury-mainnet.ckbapp.dev/ ) for mainnet mercury, so you can save much time for syning data. 
3. Run a test to create a cheque cell. Test code in [java](https://github.com/nervosnetwork/ckb-sdk-java/blob/455f866c6faf5f63c781e74edefac5f7426a4019/ckb-mercury-sdk/src/test/java/mercury/ActionTest.java#L81) or [go](https://github.com/nervosnetwork/ckb-sdk-go/blob/b7b9eff4f48fb2f3020cb31c630eea17cbc77e81/mercury/example/action_example.go#L53) .

### CODE REVIEW

**Review by quake — APPROVED** [review]

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 1]
```diff
@@ -247,6 +247,76 @@ and the `out_point` of `anyone_can_pay` is
...
+
+cheque script is for **lock script**:
+
+- Lina
+
+| parameter   | value                                                                |
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0xe4d4ecc6e5f9a059bf2f7a82cca292083aebc0c421566a52484fe2ec51a9fb0c` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0x04632cc459459cf5c9d384b43dee3e36f542a464bdd4127be7d6618ac6f8d268` |
```
Comment (doitian) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:262]: The output cells 0 in this tx has already been spent. Also the cell has no the type script.

https://explorer.nervos.org/transaction/0x04632cc459459cf5c9d384b43dee3e36f542a464bdd4127be7d6618ac6f8d268

**Review by doitian — CHANGES_REQUESTED** [review]
Please provide instructions of how to verify the information is correct.

- How to check `code_hash`, `hash_type` and dep group content.

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md (hunk 1 above)
Comment (rev-chaos) [reply rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:262]: This output cell is unspent. See the first transaction in https://explorer.nervos.org/address/ckb1qyqlgpvldxaum4w9pqd9cpkmkem6e3w0mqtqqren4v

Comment (rev-chaos) [reply rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:262]: As I can see, the type of acp dep_group cell is also null.

**Comment by rev-chaos** [comment]
> Please provide instructions of how to verify the information is correct.
> 
> * How to check `code_hash`, `hash_type` and dep group content.

You can send and claim a cheque cell in the Lina by [mercury](https://github.com/nervosnetwork/mercury) with mainnet configuration with the rightful cheque settings written in this RFC.

A theoretical way to verify:
1. The code_hash of cheque script is `0x00000000000000000000000000000000000000000000000000545950455f4944`. See the cheque deploy transaction for [mainnet](https://explorer.nervos.org/transaction/0x0a34aeea122d9795e06e185746a92e88bca0ad41b0e5842a960e5fd1d43760a6) and [testnet]((https://explorer.nervos.org/aggron/transaction/0x1b16769dc508c8349803fe65558f49aa8cf04ca495fbead42513e69e46608b6c). The deploy info is also written in this PR.
So, the binary code of cheque is consistent between mainnet and testnet.
2. The data of dep_group cell is completely obeying the dep group rules.
3. We have fully tested the cheque deploying ones on the testnet by mercury. An [example](https://explorer.nervos.org/aggron/transaction/0xed711c1354609792e9db66ce23653d07d00fb5d5c17e892b834025c53b76f227) for create cheque cell. An [example](https://explorer.nervos.org/aggron/transaction/0xe665735a28cbaefa501924f489e3464c145b7885fdfb709ef76b56e2620329ad) for claim cheque cell. 
So, according to theoretical derivation, we can make sure the cheque dep_group cell in mainnet can work.

A practical way to verify:  Use ckb-sdk v0.43.1 or above to successfully create a cheque cell.
1. Get some CKB and sUDT for test. CKB for test should above 305.
2. Set mainnet mercury URL in ckb-sdk, [java](https://github.com/nervosnetwork/ckb-sdk-java/blob/455f866c6faf5f63c781e74edefac5f7426a4019/ckb-mercury-sdk/src/test/java/mercury/constant/MercuryApiFactory.java#L8) or [go](https://github.com/nervosnetwork/ckb-sdk-go/blob/113ffa36146b34f6c294633a41ed961f0aaba1d7/mercury/example/constant/mercury_api_factory.go#L7) . We have deployed a public server here (https://mercury-mainnet.ckbapp.dev/ ) for mainnet mercury, so you can save much time for syning data. 
3. Run a test to create a cheque cell. Test code in [java](https://github.com/nervosnetwork/ckb-sdk-java/blob/455f866c6faf5f63c781e74edefac5f7426a4019/ckb-mercury-sdk/src/test/java/mercury/ActionTest.java#L81) or [go](https://github.com/nervosnetwork/ckb-sdk-go/blob/b7b9eff4f48fb2f3020cb31c630eea17cbc77e81/mercury/example/action_example.go#L53) .

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md (hunk 1 above)
Comment (doitian) [reply rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:262]: Got it. I thought it was the cheque code cell.

**Review by doitian — APPROVED** [review]

**Pull request merged**